layouter/static/layouter/css/layouter.css

```diff
@@ -70,4 +70,12 @@
 .full-width:before {
   content: "\e906";
 }
-
+@media only screen and (max-width: 480px) {
```

Comment:
Probably we should use the bootstrap media queries for that. To make it more clear to the user, I think it would help to add the widths to the corresponding help texts.